cloud/ec2.go

```diff
@@ -1369,6 +1384,18 @@ func (m *ec2Manager) TimeTilNextPayment(host *host.Host) time.Duration {
 	return timeTilNextEC2Payment(host)
 }
 
+// CleanupIP disassociates the IP address from the host's network interface and
+// releases the IP address back into the IPAM pool.
+func (m *ec2Manager) CleanupIP(ctx context.Context, h *host.Host) error {
+	if err := disassociateIPAddressForHost(ctx, m.client, h); err != nil {
+		return errors.Wrapf(err, "disassociating host IP address with association ID '%s'", h.IPAssociationID)
+	}
+	if err := releaseIPAddressForHost(ctx, m.client, h); err != nil {
+		return errors.Wrapf(err, "releasing host IP address with allocation ID '%s'", h.IPAllocationID)
+	}
+	return nil
```

Comment:
Small nit (unless you see this func expanding), could we just `return errors.Wrapf(releaseIPAddressForHost(ctx, m.client, h), "releasing host IP address with allocation ID '%s'", h.IPAllocationID)?` (although it might be wordy).

Also, should the helper functions just wrap the error? It looks like they just validate, call actual func. I think it would make sense if they wrapped the error so every usage of them have consistent wrapping.

Reply:
Yeah that's fair, I can move the wrapping around a little.